package.json

```diff
@@ -27,6 +27,8 @@
     "test-hide-eresources-func": "stripes test nightmare --run hide-eresources-func",
     "test-supplement": "stripes test nightmare --run supplement",
     "test-notes-crud": "stripes test nightmare --run notes-crud",
+    "test-package-notes": "stripes test nightmare --run package-notes",
```

Comment:
Can you also add these two tests to the `test` and `test-int` `scripts` options just after the `notes-crud` test so that they run when we do a full test-suite run?